lib/librato/rails/railtie.rb

```diff
@@ -39,6 +39,16 @@ class Railtie < ::Rails::Railtie
           end
         end
 
+        tracker = config.librato_rails.tracker
+        require_relative 'subscribers/cache' if tracker.suite_enabled?(:rails_cache)
+        require_relative 'subscribers/controller' if tracker.suite_enabled?(:rails_controller)
```

Comment:
I think this one is going to need to be more granular than this, may have to break into two subscribers?